task_processing/interfaces/event.py

```diff
@@ -35,6 +37,13 @@ class Event(PRecord):
         invariant=lambda x: (isinstance(x, PMap),
                              'task_config must inherit from PMap'),
         factory=lambda x: pmap(x) if not isinstance(x, PMap) else x)
+
+    # This fields are only applicable to event of type pod
+    task_config_per_task_id = field(type=PMap)
```

Comment:
I don't really like this. The whole point in the interface is that its platform generic, and yet you're adding more keys here. Why do you need to return all the task_configs for tasks in the pod if you're issuing a pod specific event? same with ``mesos_status_per_task_id``